examples/vision/aodnet.py

```diff
@@ -0,0 +1,354 @@
+"""
+Title: An All-in-One Network for Dehazing and Beyond
+Author: [Soumik Rakshit](https://github.com/soumik12345)
+Date created: 2021/04/10
+Last modified: 2021/04/10
+Description: Train a lightweight Deep CNN for image denhazing.
```

Comment:
Suggestion:

- Use sentence case, refactor

```suggestion
Description: Dehaze images with a lightweight deep convolutional neural network.
```

("AOD-Net" is defined later in the intro)